src/main/java/com/ty/password/dto/User.java

```diff
@@ -1,5 +1,49 @@
 package com.ty.password.dto;
 
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+@EnableWebMvc
```

Comment:
Its Entity class please change to @Entity

Reply:
yup